iati/custom_middleware.py

```diff
@@ -27,7 +27,11 @@ def __call__(self, request):
         self.path_parts = self.remove_language_code(valid_path_parts)
         self.is_download = "downloads" in self.path_parts
         self.stripped_path = '/'.join(self.path_parts)
+        full_url = request.build_absolute_uri()
+        domain = urllib.parse.urlparse(full_url).netloc
 
+        if domain == 'reference.iatistandard.org':
```

Comment:
I think we want to check here for both domain and the path. Because I interpreted the ticket to only redirect `reference.iatistandard.org` and `reference.iatistandard.org/` exclusively. I think catching this domain here would also redirect `reference.iatistandard.org/anything-else` which I don't think we want.